client-java/src/unit-tests/resources/logback.xml

```diff
@@ -29,6 +29,8 @@
     </root>
 
     <logger name="uk.gov.gchq.palisade.client" level="DEBUG"/>
+    <logger name="uk.gov.gchq.palisade.client.abc.impl" level="DEBUG"/>
+    <logger name="uk.gov.gchq.palisade.client.internal.download" level="DEBUG"/>
```

Comment:
Is this needed beyond setting the parent-path to debug?

Reply:
This is very useful to filter out framework logging. An example is when you just wish to see the web socket loop in action. Quite often I would set one or more of those to INFO. It's useful to have them around.

Reply:
But why do that from the logback xml file in unit-tests instead of a cli param?
If these were different, if these were not unit-tests, etc. i could see why, but in this case, there's no real benefit over just using the root-level <logger "...palisade.client" ,,,> tag

Reply:
I've removed all the loggers except the main one for client. If needed another dev will just have to add them back in.